boot/mbed/include/flash_map_backend/secondary_bd.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2015 Runtime Inc
```

Comment:
Good catch on this one, I'll remove those lines since secondary_bd is mbed-specific.
For the other files that are based on existing configs/code for other platforms in the mcuboot repo, I kept the copyrights in the original files